env/parameters.schema.json

```diff
@@ -39,6 +40,11 @@
           "title": "Pipeline bot Git username",
           "description": "The Git user that will perform git operations inside a pipeline. It should be a user within the Git organisation/owner where environment repositories will live. This is normally a bot."
         },
+        "email": {
+          "type": "string",
+          "title": "Pipeline bot Git email address",
+          "description": "The email address of the Git user that will perform git operations inside a pipeline."
+        },
```

Comment:
You probably need to ask for name as well e.g. my setup is:

username: `pmuir`
name: `Pete Muir`
email: `redacted`

Reply:
This is for the bot user though, which wouldn't have a name? and I don't think it's needed as I'm setting GIT_AUTHOR_NAME and GIT_AUTHOR_EMAIL.

As this is the bot user it would of been nice to only ask the email question if `gitops: true` but I couldn't see that you could do conditionals in this schema?

Reply:
I see your point about the name. I guess it's reasonable until someone complains. Conditionals are possible, but not on external elements like requirements right now :-(